### PR TITLE
Itos 122

### PR DESCRIPTION
**Pull request by areklalo**

### CODE REVIEW

File: sendReports.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export MYSQL_HOST
+export MYSQL_ROOT_PASSWORD
+
+USERNAME=admin;
+POST_TEMPLATE=$(cat /postTemplate.json)
+
+processFile() {
+   REPORT_CONTENT=$(envsubst "`printf '${%s} ' $(bash -c "compgen -A variable")`"  < $1)
+   REPORT_CONTENT="${REPORT_CONTENT//\"/\\\"}"
+   REPORT_NAME=$(echo $2 | sed 's/\([A-Z][^A-Z]\)/ \1/g')
```
Comment (wojtekbu) [inline sendReports.sh]: Is this regex exclude small letter?

File: sendReports.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+   POST_CONTENT=$(printf "$POST_TEMPLATE" "$REPORT_NAME" "$REPORT_CONTENT")
+
+   curl -X POST -H "Content-Type: application/json" -u "$USERNAME:$DHIS_ADMIN_PASSWORD" \
+   "http://localhost:8080/api/reports" -d "$(echo $POST_CONTENT)"
```
Comment (wojtekbu) [inline sendReports.sh]: Should it be the permalink to localhost?

File: sendReports.sh (hunk 1 above)
Comment (areklalo) [reply sendReports.sh]: This regex change camel case to the sentence. Example "sampleReport" will change to "sample Report".

File: sendReports.sh (hunk 2 above)
Comment (areklalo) [reply sendReports.sh]: Yes because this script is running only on the ohie-fr.

File: sendReports.sh (hunk 1 above)
Comment (wojtekbu) [reply sendReports.sh]: OK, it's fine for a set of filenames that we've defined before.

File: sendReports.sh (hunk 2 above)
Comment (wojtekbu) [reply sendReports.sh]: Ok

**Pull request merged**